std/random.d

```diff
@@ -1614,6 +1614,73 @@ if (!is(T == enum) && (isIntegral!T || isSomeChar!T))
 }
 
 /**
+Generates a random `bool` from a uniformly-distributed number in the range
+$(D [uint.min, uint.max]). A new uniformly-distributed number is generated
+every 32 calls of `uniform!bool`. In multithreaded environments, each `thread`
+has it's own uniformly-distributed number. If no random number generator is
```

Comment:
No need for too much detail/commitment, just specify that the underlying random integral generation is batched.